docs/experiment/sdks/jvm-sdk.md

```diff
@@ -0,0 +1,441 @@
+        if (variant?.value == "on") {
+            // Flag is on
+        } else {
+            // Flag is off
+        }
+        ```
+
+    === "Java"
+
+        ```java
+        // (1) Initialize the remote evaluation client with a server deployment key.
+        RemoteEvaluationClient experiment =
+                Experiment.initializeRemote("<DEPLOYMENT_KEY>");
+
+        // (2) Fetch variants for a user
+        ExperimentUser user = ExperimentUser.builder()
+                .userId("user@company.com")
+                .deviceId("abcdefg")
+                .userProperty("premium", true)
+                .build();
+        Map<String, Variant> variants;
+        try {
+            variants = experiment.fetch(user).get();
+        } catch (Exception e) {
+            e.printStackTrace();
+            return;
+        }
+
+        // (3) Access a flag's variant
+        Variant variant = variants.get("<FLAG_KEY>");
+        if (Variant.valueEquals(variant, "on")) {
+            // Flag is on
+        } else {
+            // Flag is off
+        }
+        ```
+
+### Initialize remote
+
+The SDK client should be initialized in your server on startup. The [deployment key](../general/data-model.md#deployments) argument passed into the `apiKey` parameter must live within the same project that you are sending analytics events to.
+
+=== "Kotlin"
+
+    ```kotlin
+    fun initializeRemote(
+        apiKey: String,
+        config: RemoteEvaluationConfig = RemoteEvaluationConfig()
+    ): RemoteEvaluationClient
+    ```
+
+=== "Java"
+
+    ```java
+    @Nonnull
+    public RemoteEvaluationClient initializeRemote(
+        @Nonnull String apiKey,
+        @Nonnull RemoteEvaluationConfig config
+    );
+
+| Parameter | Requirement | Description |
+| --- | --- | --- |
+| `apiKey` | required | The [deployment key](../general/data-model.md#deployments) which authorizes fetch requests and determines which flags should be evaluated for the user. |
+| `config` | optional | The client [configuration](#configuration) used to customize SDK client behavior. |
+
+=== "Kotlin"
+
+    ```kotlin
+    val experiment = Experiment.initializeRemote("<DEPLOYMENT_KEY>")
+    ```
+
+=== "Java"
+
+    ```java
+    RemoteEvaluationClient experiment = Experiment.initializeRemote("<DEPLOYMENT_KEY>");
+    ```
+
+#### Configuration
+
+The SDK client can be configured on initialization.
+
+???config "Configuration Options"
+    | <div class="big-column">Name</div> | Description | Default Value |
+    | --- | --- | --- |
+    | `debug` | Set to `true` to enable debug logging. | `false` |
+    | `serverUrl` | The host to fetch flag configurations from. | `https://api.lab.amplitude.com` |
+    | `fetchTimeoutMillis` |  The timeout for fetching variants in milliseconds. This timeout only applies to the initial request, not subsequent retries | `500` |
+    | `fetchRetries` | The number of retries to attempt if a request to fetch variants fails. | `1` |
```

Comment:
default retry once?

Reply:
Yeah, I think this is a better default. This is what we've landed on using in thunder for example.